Makefile

```diff
@@ -31,7 +31,7 @@ ineffassign: ## Run ineffassign checker
 	./ineffassign.sh
 
 shellcheck: ## Run shellcheck
-	shellcheck **/*.sh
+	shellcheck *.sh */*.sh
```

Comment:
Probably `find . -exec` would also work here

Reply:
Maybe, but we can have Python scripts that are executable too, so I prefer to find them using the extension (probably using `find . -name "*.sh"` could be better...